animal_crossing.py

```diff
@@ -0,0 +1,73 @@
+import sqlite3
+
+import config
+
+if config.bot.acnh_db is not None:
+	db = sqlite3.connect(config.bot.acnh_db)
+
+def friend_code(cmd):
+	if not cmd.args:
+		_fc_list_all(cmd)
+		return
+
+	split = cmd.args.split(' ', 1)
+	subcmd = split[0]
+	if subcmd == 'add':
+		_fc_add_or_update(cmd, split)
+	elif subcmd == 'remove':
+		_fc_remove(cmd)
+	else:
+		_fc_find(cmd, split)
+
+def _fc_add_or_update(cmd, split):
+	fc = split[1]
+	sender = cmd.sender
+	cur = db.execute('''
+	SELECT id FROM friend_code WHERE id=?
+	''', (sender['id'],))
+
+	try:
+		cur.fetchone()
+		db.execute('''
+		UPDATE friend_code SET code=? WHERE id=?
+		''', (fc, sender['id']))
+		db.commit()
+		cmd.reply('Updated friend code!')
+	except:
```

Comment:
bare `except:` is pretty surprising since it's not short for `except Exception:` like you expect. assuming you were gonna do this later but I think this would be `except sqlite3.OperationalError:`?

Reply:
oh definitely was thinking it was `except Exception` lol